modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-enabling-workload-partitioning_{context}"]
+= Enabling workload partitioning
+
+A key feature to enable as part of single node installation is workload partitioning.
+This limits the cores allowed to run platform services maximizing the CPU core for application payloads. No customization is required for the configuration.
+Use the following file to enable workload partitioning:
```

Comment:
We should repeat the note from earlier in the document that workload partitioning must be configured at cluster installation time.

Reply:
added